cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2211,6 +2216,7 @@ describe('CourseOutlinePage', function() {
                 outlinePage.$('.outline-item .outline-subsection .configure-button').click();
                 expect($('#due_in').val()).toBe('3');
                 expect($('#grading_type').val()).toBe('Lab');
+                expect($('#due_num_weeks_projected.message').css('display')).not.toBe('none');
```

Comment:
Out of curiosity, any reason you sometimes check values and other times check the display being none or not?

Reply:
yeah dates worked differently in different browsers (ex. Firefox vs. Google Chrome) and trying to figure out all the different cases for a single date didn't seem worth the time, so I just check if it is not none